src/components/MonitorListFilter.vue

```diff
@@ -191,52 +194,63 @@ export default {
         this.getExistingTags();
     },
     methods: {
+        getActiveFilters: function () {
+            const filters = Object.fromEntries(
```

Comment:
Not a js-magician on my side.
What is happeing here?

Reply:
JS magic. Yeah, should've been:
```js
const filters = this.$router.currentRoute.value.queey
```